fork-runner/src/main/java/com/shazam/fork/runner/listeners/RetryListener.java

```diff
@@ -26,36 +26,32 @@
 public class RetryListener extends NoOpITestRunListener {
     private static final Logger logger = LoggerFactory.getLogger(RetryListener.class);
     private final Device device;
-    private final TestCaseEvent currentTestCaseEvent;
-    private final TestRetryer testRetryer;
+    private final FailedTestScheduler failedTestScheduler;
     private final Pool pool;
     private final DeviceTestFilesCleaner deviceTestFilesCleaner;
-    private TestIdentifier startedTest;
-    private TestIdentifier failedTest;
+    private TestCaseEvent startedTest;
+    private TestCaseEvent failedTest;
```

Comment:
Why is this changed from TestIdentifier to TestCaseEvent?

Reply:
I refactored FailedTestScheduler to use our custom model `TestCaseEvent` instead of `TestIdentifier` which is coming from ddmlib.

Reply:
🥇